README.md

```diff
@@ -0,0 +1,104 @@
+Project Plan google doc link (https://docs.google.com/document/d/1QTFmvcbl0Ds0DxBxlxneavytynoTvsAw92YnZO4pCj4/edit?pli=1&tab=t.0)
```

Comment:
You don't need to refactor your readme as a separate file. You can do this separately